### ISSUE TITLE
Pilot fails with "ERROR   : bulk launch failed" on Stampede 2

### ISSUE THREAD

**Issue by SrinivasMushnoori** [radical-cybertools/saga-python#693]
Appears to be a SAGA issue. There are no logs on the agent side. The VE sandbox is never created. On the client side, I see the following in `pmgr.0000.launching.0.child.log`  : 

```
2018-11-19 10:45:12,696: pmgr.0000.launching.0.child: pmgr.0000.launching.0           : MainThread     : ERROR   : bulk launch failed
Traceback (most recent call last):
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/radical/pilot/pmgr/launching/default.py", line 488, in work
    self._start_pilot_bulk(resource, schema, pilots)
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/radical/pilot/pmgr/launching/default.py", line 677, in _start_pilot_bulk
    js = rs.job.Service(js_ep, session=self._session)
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/saga/job/service.py", line 115, in __init__
    url, session, ttype=_ttype)
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/saga/base.py", line 104, in __init__
    self._init_task = self._adaptor.init_instance (adaptor_state, *args, **kwargs)
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/saga/adaptors/cpi/decorators.py", line 57, in wrap_function
    return sync_function (self, *args, **kwargs)
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/saga/adaptors/slurm/slurm_job.py", line 295, in init_instance
    self._open()
  File "/home/scm177/VirtualEnvs/Env_RepEx/local/lib/python2.7/site-packages/saga/adaptors/slurm/slurm_job.py", line 377, in _open
    self._version = StrictVesrion(out.split()[1].strip())
NameError: global name 'StrictVesrion' is not defined
```

`radical-stack` is 
```
  python               : 2.7.14
  pythonpath           : 
  virtualenv           : /home/scm177/VirtualEnvs/Env_RepEx

  radical.analytics    : v0.45.2-104-ge615742@feature-entk-integration
  radical.entk         : 0.7.10-0.7.10@devel
  radical.pilot        : 0.50.19-v0.50.19@master
  radical.utils        : 0.50.1-v0.50.1-2-gfebbcd0@devel
  saga                 : 0.50.0-v0.50.0-3-gcbff7b1@devel
```

Logs are [here](https://github.com/radical-cybertools/saga-python/files/2596025/logs.tar.gz)

**Comment by iparask** [comment]
Please pull devel again. This was fixed by #691

**Comment by SrinivasMushnoori** [comment]
devel seems to downgrade from 0.50.3 to 0.50.0, unsure if #688 will resurface?
Or is this just incorrect version reporting?

**Comment by iparask** [comment]
The error you saw was because there was a typo on the fix for #688, which was closed by #689.

**Comment by SrinivasMushnoori** [comment]
I see, thanks for clarifying. Just tested, that particular error goes away when I pulled again from devel. Closing.
